kura/org.eclipse.kura.ble.provider/src/main/java/org/eclipse/kura/internal/ble/BluetoothLeServiceImpl.java

```diff
@@ -21,36 +21,42 @@
 import org.eclipse.kura.bluetooth.le.BluetoothLeService;
 import org.osgi.service.component.ComponentContext;
 
-import tinyb.BluetoothManager;
+import org.freedesktop.dbus.exceptions.DBusException;
+
+import com.github.hypfvieh.bluetooth.DeviceManager;
+import com.github.hypfvieh.bluetooth.wrapper.BluetoothAdapter;
 
 public class BluetoothLeServiceImpl implements BluetoothLeService {
 
     private static final Logger logger = LogManager.getLogger(BluetoothLeServiceImpl.class);
 
-    private BluetoothManager bluetoothManager;
+    private DeviceManager bluetoothManager;
 
```

Comment:
It's worth to change the variable name from `bluetoothManager` to `deviceManager`?

Reply:
Probably makes sense :)